FrameworkSystem/scripts/dirac-proxy-init.py

```diff
@@ -19,237 +19,244 @@
     else:
-      gLogger.notice( "Proxy generated:" )
-      gLogger.notice( resultProxyInfoAsAString[ 'Value' ] )
+      gLogger.notice("Proxy generated:")
+      gLogger.notice(resultProxyInfoAsAString['Value'])
     if self.__uploadedInfo:
-      gLogger.notice( "\nProxies uploaded:" )
+      gLogger.notice("\nProxies uploaded:")
       maxDNLen = 0
       maxGroupLen = 0
       for userDN in self.__uploadedInfo:
-        maxDNLen = max( maxDNLen, len( userDN ) )
-        for group in self.__uploadedInfo[ userDN ]:
-          maxGroupLen = max( maxGroupLen, len( group ) )
-      gLogger.notice( " %s | %s | Until (GMT)" % ( "DN".ljust( maxDNLen ), "Group".ljust( maxGroupLen ) ) )
+        maxDNLen = max(maxDNLen, len(userDN))
+        for group in self.__uploadedInfo[userDN]:
+          maxGroupLen = max(maxGroupLen, len(group))
+      gLogger.notice(" %s | %s | Until (GMT)" % ("DN".ljust(maxDNLen), "Group".ljust(maxGroupLen)))
       for userDN in self.__uploadedInfo:
-        for group in self.__uploadedInfo[ userDN ]:
-          gLogger.notice( " %s | %s | %s" % ( userDN.ljust( maxDNLen ),
-                                              group.ljust( maxGroupLen ),
-                                              self.__uploadedInfo[ userDN ][ group ].strftime( "%Y/%m/%d %H:%M" ) ) )
+        for group in self.__uploadedInfo[userDN]:
+          gLogger.notice(" %s | %s | %s" % (userDN.ljust(maxDNLen),
+                                            group.ljust(maxGroupLen),
+                                            self.__uploadedInfo[userDN][group].strftime("%Y/%m/%d %H:%M")))
 
-  def checkCAs( self ):
+  def checkCAs(self):
     if not "X509_CERT_DIR" in os.environ:
-      gLogger.warn( "X509_CERT_DIR is unset. Abort check of CAs" )
+      gLogger.warn("X509_CERT_DIR is unset. Abort check of CAs")
       return
-    caDir = os.environ[ "X509_CERT_DIR" ]
-     # In globus standards .r0 files are CRLs. They have the same names of the CAs but diffent file extension
-    searchExp = os.path.join( caDir, "*.r0" )
-    crlList = glob.glob( searchExp )
+    caDir = os.environ["X509_CERT_DIR"]
+    # In globus standards .r0 files are CRLs. They have the same names of the CAs but diffent file extension
+    searchExp = os.path.join(caDir, "*.r0")
+    crlList = glob.glob(searchExp)
     if not crlList:
-      gLogger.warn( "No CRL files found for %s. Abort check of CAs" % searchExp )
+      gLogger.warn("No CRL files found for %s. Abort check of CAs" % searchExp)
       return
-    newestFPath = max( crlList, key=os.path.getmtime )
-    newestFTime = os.path.getmtime( newestFPath )
-    if newestFTime > ( time.time() - ( 2 * 24 * 3600 ) ):
-      # At least one of the files has been updated in the last 2 days
+    newestFPath = max(crlList, key=os.path.getmtime)
+    newestFTime = os.path.getmtime(newestFPath)
+    if newestFTime > (time.time() - (28 * 24 * 3600)):
+      # At least one of the files has been updated in the last 28 days
```

Comment:
this was intentionally changed to 2 days by @sfayer in https://github.com/DIRACGrid/DIRAC/pull/3560

Reply:
Argh, thanks ! Very nicely spotted ! 
I fear that with the insane mess I had to go through starting from Milosz PR to rebase and merge, I might have lost a few things :(